openapi.yaml

```diff
@@ -4783,10 +4888,29 @@ components:
            that have a process graph included.
            It is RECOMMENDED to log the namespace selected by the back-end for debugging purposes.
         * `backend`: Uses exclusively the pre-defined processes listed at `GET /processes`.
-        * `user`: Uses exclusively the user-defined processes listed at `GET /process_graphs`.
+        * `user`: Uses exclusively the user-defined processes listed at `GET /processes/@{user_id}`
+          with `{user_id}` being the user id of the authenticated user. This is an alias for 
+          directly specifying `@{user_id}` as namespace.
 
         If multiple processes with the same identifier exist, Clients SHOULD
         inform the user that it's recommended to select a namespace.
+    process_namespace:
+      type: string
+      description: |-
+        The namespace for a process.
+
+        The API only defines a single type of predefined namespaces. That is for
+        user-defined processes stored at the back-end and defines a distinct
+        namespace for each individual user. They are prescribed with a leading
+        `@` followed by the unique identifier of a user.
+
+        The namespace `user` is an alias for `@` + user id of the authenticated user.
+        It can be used to retrieve the user-defined processes of the authenticated user.
+        
+        The namespace `backend` is an alias for predefined processes.
+
+        Back-end implementations MAY implement other namespaces that don't
+        conflict with any of the namespaces mentioned above.
```

Comment:
see discussion at https://github.com/Open-EO/openeo-api/issues/310#issuecomment-740567496

I think the namespace "format" should be more flexible/generic than: `@` is for per-user namespaces

Reply:
And I disagree in #310 ;-)

Reply:
In the meantime I'm fine with more flexibility, but on the other hand we should likely restrict the allowed characters:
https://github.com/Open-EO/openeo-api/issues/478